frontend/src/App.tsx

```diff
@@ -47,7 +48,9 @@ function AppWithRedux(props: React.PropsWithChildren<{}>) {
 
   return (
     <I18nextProvider i18n={i18n}>
-      <ThemeProviderNexti18n theme={muiTheme}>{props.children}</ThemeProviderNexti18n>
+      <MonacoEditorProvider>
```

Comment:
Can this MonacoEditorProvider be moved to be closer where it's used?

At some point I'd like the monaco editor to be lazy loaded, but the code is all a bit tangled at the moment to do this easily. For this reason it would be good if it's possible to move this closer to where the editor is used.